### ISSUE TITLE
Default font (AFM fonts)

### ISSUE THREAD

**Issue by knownasilya** [rkusa/pdfjs#20]
Would love a default font so that it doesn't have to be set every time.

**Comment by rkusa** [comment]
Are you using the upcoming 0.8 version or the NPM version?

**Comment by knownasilya** [comment]
@rkusa using 0.8 from github.

**Comment by rkusa** [comment]
The font settings is inherited through all elements:

```js
var doc = new pdfjs.Document({
  font:    new pdfjs.TTFFont(buffer)
})
```

The above code should work like a default font. Does this work for you?

**Comment by knownasilya** [comment]
@rkusa yes, but I meant having a font set if the user doesn't specify one. There must be a away to get system fonts.

**Comment by rkusa** [comment]
Yes, there is a way. PDF has about 20 integrated default fonts, called AFM fonts. However, you still need access to them for layout calculation (e.g. character widths).

I've this feature on my todo list. But as a plugin, since they must be bundled into the source code, and I'd like to keep the overall size a low as possible for client-side usage.

**Comment by rkusa** [comment]
Implemented in `2.0.0-alpha.1`, as shown [here](https://github.com/rkusa/pdfjs/tree/master/docs#new-pdffontarg)
